includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,404 @@
+				],
+			]
+		);
+
+		$this->add_control(
+			'card_heart_diamond_color',
+			[
+				'label' => esc_html__( 'Hearts and Diamonds Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-card-heart, {{WRAPPER}} .elementor-playing-card-diamond' => 'color: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->add_control(
+			'card_spade_club_color',
+			[
+				'label' => esc_html__( 'Spades and Clubs Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-card-spade, {{WRAPPER}} .elementor-playing-card-club' => 'color: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->end_controls_section();
+
+		$this->start_controls_section(
+			'section_card_cover_styles',
+			[
+				'label' => esc_html__( 'Card cover', 'elementor' ),
+				'tab' => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_control(
+			'card_cover_background',
+			[
+				'label' => esc_html__( 'Background Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-card-hidden::before' => 'background-color: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->add_control(
+			'card_cover_icon_color',
+			[
+				'label' => esc_html__( 'Icon Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-card-hidden::before' => 'color: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->end_controls_section();
+	}
+
+	/**
+	 * Render playing cards widget output on the frontend.
+	 *
+	 * Written in PHP and used to generate the final HTML.
+	 *
+	 * @since 3.12.1
+	 * @access protected
+	 */
+	protected function render() {
+		$cards = $this->get_settings_for_display( 'cards_list' );
+		$hover_animation = $this->get_settings_for_display( 'hover_animation' );
+
+		$this->add_render_attribute( 'list', 'class', 'elementor-playing-cards elementor-playing-cards-' . $hover_animation );
+		$this->add_render_attribute( 'hiding-button', 'aria-label', esc_html__( 'Hide playing cards type and value', 'elementor' ) );
+
+		$types = [
+			'spade' => '♠',
+			'diamond' => '♦',
+			'heart' => '♥',
+			'club' => '♣',
+		];
+		?>
+		<div class="elementor-playing-cards-wrapper" tabindex="0">
+			<span class="elementor-screen-only" role="heading">
+				<?php echo esc_html__( 'Playing Cards', 'elementor' ); ?>
+			</span>
+
+			<div <?php $this->print_render_attribute_string( 'list' ); ?>>
+				<?php
+				foreach ( $cards as $card ) {
+					$id = $card['_id'];
+					$type = $card['card_type'];
+					$value = $card['card_value'];
+
+					$this->add_render_attribute( 'card-' . $id, 'class', sprintf( 'elementor-playing-card elementor-playing-card-%s elementor-playing-card-%s', $id, $type ) );
+
+					$this->add_render_attribute( 'card-' . $id . '-heading', 'data-type', $type );
+					$this->add_render_attribute( 'card-' . $id . '-heading', 'data-value', $value );
```

Comment:
@pkniazevych If it's the same element, you can make one function call and pass an array:

$this->add_render_attribute( 'card-' . $id . '-heading', [
...
] );